tutorials/getting_started/getting_started.md

```diff
@@ -0,0 +1,181 @@
+    "cuda_device": -1
+  }
+```
+
+The `num_epochs` parameter just specifies how many passes through the dataset we want to train on.
+On a recent Macbook each epoch of this model on this dataset takes about 30 seconds, so 20 will take about 10 minutes.
+The `serialization` prefix is the path where the model's vocabulary and checkpointed weights will be saved.
+And if you have a GPU you can change `cuda_device` to 0 to use it.
+
+Change any of those if you want to, and then run
+
+```
+allennlp/run train tutorials/getting_started/simple_tagger.json
+```
+
+It will log all of the parameters it's using and then display the progress and results of each epoch:
+
+```
+2017-08-15 11:37:53,030 - INFO - allennlp.training.trainer - Epoch 5/20
+accuracy: 0.48, accuracy_top3: 0.63, loss: 2.19 ||: 100%|##########| 477/477 [00:25<00:00, 21.68it/s]
+accuracy: 0.50, accuracy_top3: 0.64, loss: 2.27 ||: 100%|##########| 50/50 [00:00<00:00, 53.72it/s]
+2017-08-15 11:38:19,609 - INFO - allennlp.training.trainer - Training accuracy : 0.477715    Validation accuracy : 0.501286
+2017-08-15 11:38:19,610 - INFO - allennlp.training.trainer - Training accuracy3 : 0.631720    Validation accuracy3 : 0.642796
+2017-08-15 11:38:19,610 - INFO - allennlp.training.trainer - Training loss : 2.187194    Validation loss : 2.265297
+2017-08-15 11:38:19,617 - INFO - allennlp.training.trainer - Best validation performance so far. Copying weights to /tmp/tutorials/getting_started/best.th'.
+```
+
+Here `accuracy` measures how often our model predicted the "correct" part of speech tag as most probable,
+while `accuracy3` measures how often the correct tag was one of the _three_ most probable.
+(`loss` measures [cross entropy](https://en.wikipedia.org/wiki/Cross_entropy), which you probably don't care about
+ other than to make sure that it's mostly decreasing during training.)
+
+After 20 epochs we see we see
+
+```
+2017-08-15 13:22:01,591 - INFO - allennlp.training.trainer - Epoch 20/20
+accuracy: 0.85, loss: 0.58, accuracy3: 0.92 ||: 100%|##########| 477/477 [00:25<00:00, 19.75it/s]
+accuracy: 0.71, loss: 1.38, accuracy3: 0.83 ||: 100%|##########| 50/50 [00:01<00:00, 47.64it/s]
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training accuracy : 0.849197    Validation accuracy : 0.714408
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training loss : 0.577224    Validation loss : 1.378879
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training accuracy3 : 0.924702    Validation accuracy3 : 0.831475
+2017-08-15 13:22:28,222 - INFO - allennlp.training.trainer - Best validation performance so far. Copying weights to /tmp/tutorials/getting_started/best.th'.
+```
+
+This means that 71% of the time our model predicted the correct tag on the validation dataset,
+and 83% of the time the correct tag was in the model's "top 3".
+(Not ground-breaking performance, but this is a pretty simple model, and
```

Comment:
Is performance low because it's a simple model, our training data is small, or the number of epochs we're using is small?